gtsfm/utils/graph.py

```diff
@@ -119,3 +126,60 @@ def extract_cyclic_triplets_from_edges(edges: List[Tuple[int, int]]) -> List[Tup
                 triplets.add(cycle_nodes)
 
     return list(triplets)
+
+
+def draw_graph_topology(
```

Comment:
`draw_graph_topology` seems a little ambiguous; maybe `draw_view_graph` would be more descriptive?

Reply:
Good point -- I renamed to `draw_view_graph_topology()`, let me know if that works.